add-on/src/lib/ipfs-proxy/index.js

```diff
@@ -5,6 +5,8 @@ const browser = require('webextension-polyfill')
 const { createProxyServer, closeProxyServer } = require('ipfs-postmsg-proxy')
 const AccessControl = require('./access-control')
 
+// These are the functions that require an allow/deny decision by the user.
+// All other exposed IPFS functions are available to call without authorization.
```

Comment:
Perhaps one for a next PR, but ongoing maintenance would be simpler if we flip this to a whitelist of methods that can be called without access control... as the ipfs api changes we can jump to the latest release and then figure out later if a new api should be whitelisted.